### PR TITLE
Surge Catalogue Page

### PR DESCRIPTION
**Pull request by samshara**
- Add emergency catalog page
- Add catalogue general tab

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/views/AssessmentCell/index.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const strings = useTranslation(i18n);
+
+    const navigate = useNavigate();
+
+    const handleBackButtonClick = useCallback(() => {
+        navigate(-1);
+    }, [navigate]);
```
Comment (tnagorra) [inline src/views/AssessmentCell/index.tsx:21]: @frozenhelium What should be the behavior of the back button? Should we just go back on history

File: src/App/routes.tsx [hunk 2]
```diff
@@ -706,6 +706,79 @@ const allSurgeAlerts = customWrapRoute({
...
+        visibility: 'anything',
+    },
+});
+
+const catalogueIndex = customWrapRoute({
+    parent: surgeCatalogue,
+    index: true,
+    component: {
+        eagerLoad: true,
+        render: Navigate,
```
Comment (tnagorra) [inline src/App/routes.tsx]: @frozenhelium do we need to navigate or just display catalogueService page in the index route

**Review by tnagorra — APPROVED** [review]

File: src/App/routes.tsx (hunk 2 above)
Comment (frozenhelium) [reply src/App/routes.tsx]: We probably need to navigate

File: src/views/CatalogueBasecampIndex/i18n.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "namespace": "surge",
```
Comment (tnagorra) [inline src/views/CatalogueBasecampIndex/i18n.json:2]: @frozenhelium @samshara Should we separate out the namespace per view?

File: src/views/BasecampFacilityManagement/index.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const strings = useTranslation(i18n);
+
+    const navigate = useNavigate();
+
+    const handleBackButtonClick = useCallback(() => {
+        navigate(-1);
```
Comment (tnagorra) [inline src/views/BasecampFacilityManagement/index.tsx:18]: @frozenhelium Should this be the behavior for "go back" links?

File: src/views/BasecampEruMedium/index.tsx [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+        <Container
+            className={styles.basecampEruMedium}
+            heading={strings.basecampEruMediumTitle}
+            childrenContainerClassName={styles.content}
+            icons={(
+                <IconButton
+                    name={undefined}
+                    onClick={handleBackButtonClick}
+                    ariaLabel="go-back"
+                    variant="tertiary"
```
Comment (tnagorra) [inline src/views/BasecampEruMedium/index.tsx:32]: If we are using icon buttons, let's alsway pass title prop

File: src/views/BasecampEruLarge/index.tsx [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+        <Container
+            className={styles.basecampEruLarge}
+            heading={strings.basecampEruLargeTitle}
+            childrenContainerClassName={styles.content}
+            icons={(
+                <IconButton
+                    name={undefined}
+                    onClick={handleBackButtonClick}
+                    ariaLabel="go-back"
+                    variant="tertiary"
```
Comment (tnagorra) [inline src/views/BasecampEruLarge/index.tsx:32]: Let's also pass title prop

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/views/AssessmentCell/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply src/views/AssessmentCell/index.tsx:21]: Not sure, we need to check how it was implemented previously

**Review by frozenhelium — APPROVED** [review]

**Review by tnagorra — APPROVED** [review]

**Pull request merged**